packages/admin/cms-admin/src/common/cleanTypename.ts

```diff
@@ -0,0 +1,3 @@
+export function cleanTypename(data: unknown) {
+    return JSON.parse(JSON.stringify(data), (key, value) => (key === "__typename" ? undefined : value));
+}
```

Comment:
There is no need for JSON serializing just for that.

An alterative version could look like this:
```
export function cleanTypename<T>(data: T): T {
    if (Array.isArray(data)) {
        return data.map(cleanTypename) as unknown as T;
    }

    if (data !== null && typeof data === "object") {
        const result: any = {};
        for (const [key, value] of Object.entries(data)) {
            if (key !== "__typename") {
                result[key] = cleanTypename(value);
            }
        }
        return result;
    }

    return data;
}
```

Reply:
or use https://www.npmjs.com/package/omit-deep